test/canvas.test.js

```diff
@@ -947,7 +947,7 @@ describe('Canvas', function () {
       assert.ok(metrics.alphabeticBaseline > 0) // ~4-5
       assert.ok(metrics.actualBoundingBoxAscent > 0)
       // On the baseline or slightly above
-      assert.ok(metrics.actualBoundingBoxDescent <= 0)
```

Comment:
i tweaked the baseline offset, so i expect some text measurement to change slightly. Unsure if this is ok or not. Any thought anyone?